hbook/ntup/ntup_example_test.go

```diff
@@ -47,3 +48,67 @@ func ExampleNtuple_scanH2D() {
 	// XStdErr:    0.957427
 	// YStdErr:    0.957427
 }
+
+func ExampleNtuple_scanH() {
+	nt, err := ntcsv.Open(
+		"ntcsv/testdata/simple-with-header.csv",
+		ntcsv.Comma(';'),
+		ntcsv.Header(),
+		ntcsv.Columns("v1", "v2", "v3"),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer nt.DB().Close()
+	var (
+		xmin = +math.MaxFloat64
```

Comment:
I would rename `xmin` into `v1min`, `xmax` into `v1max`.
similarly for `ymin` and `v2min`.